package.json

```diff
@@ -35,19 +35,19 @@
   "dependencies": {
     "@homebound/form-state": "^2.17.0",
     "@internationalized/number": "^3.0.3",
+    "@popperjs/core": "^2.11.6",
     "@react-aria/utils": "^3.13.2",
-    "@react-hook/resize-observer": "^1.2.2",
     "change-case": "^4.1.2",
     "date-fns": "^2.28.0",
     "dompurify": "^2.3.0",
     "fast-deep-equal": "^3.1.3",
-    "framer-motion": "^4.1.11",
+    "framer-motion": "^9.0.4",
     "memoize-one": "^5.2.1",
     "react-aria": "^3.18.0",
     "react-day-picker": "8.0.7",
-    "react-popper": "^2.2.5",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0",
+    "react-popper": "^2.3.0",
+    "react-router": "^5.3.4",
+    "react-router-dom": "^5.3.4",
```

Comment:
Would love to get this up to 6 at some point as well. Or maybe even start looking at other router options... TanStack Router?